src/main/webapp/app/localci/build-queue/build-queue.component.ts

```diff
@@ -104,15 +112,31 @@ export enum FinishedBuildJobFilterStorageKey {
     selector: 'jhi-build-queue',
     templateUrl: './build-queue.component.html',
     styleUrl: './build-queue.component.scss',
+    standalone: true,
+    imports: [
+        ArtemisSharedModule,
+        ArtemisSharedComponentModule,
+        BuildJobStatisticsComponent,
+        FormDateTimePickerModule,
+        SubmissionResultStatusModule,
+        NgxDatatableModule,
+        ArtemisDataTableModule,
```

Comment:
Not sure, but the components looks quite small, are you sure that you need all these imports? Maybe removing some of them would work